eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -595,6 +605,19 @@ jobs:
       TargetFolder: $(Build.ArtifactStagingDirectory)/publishing
     displayName: Copy artifacts to Artifact Staging Directory
 
+  - ${{ if eq(parameters.targetOS, 'windows') }}:
```

Comment:
Any reason to not use CopyFiles here?

Reply:
I couldn't figure out how to rename the file with CopyFiles
on the other hand I'm not sure if there's a specific reason to name the vertical manifests `VerticalManifest.xml` instead of $(JobName).xml on creation